target_chains/solana/sdk/js/pyth_solana_receiver/src/PythSolanaReceiver.ts

```diff
@@ -489,13 +587,12 @@ export class PythSolanaReceiver {
       computeUnits: INIT_ENCODED_VAA_COMPUTE_BUDGET,
     });
 
-    postInstructions.push(
-      ...(await buildWriteEncodedVaaWithSplitInstructions(
-        this.wormhole,
-        trimmedVaa,
-        encodedVaaKeypair.publicKey
-      ))
+    const writeInstructions = await buildWriteEncodedVaaWithSplitInstructions(
+      this.wormhole,
```

Comment:
why this change of syntax here? Aren't they equivalent?

Reply:
yeah just a remnant of debugging